src/mappings/HistoryElements.ts

```diff
@@ -9,14 +17,41 @@ export async function handleHistoryElement(extrinsic: SubstrateExtrinsic): Promi
         element.address = extrinsic.extrinsic.signer.toString()
         element.timestamp = timestamp(extrinsic.block)
 
-        element.extrinsic = {
-            hash: extrinsic.extrinsic.hash.toString(),
-            module: extrinsic.extrinsic.method.section,
-            call: extrinsic.extrinsic.method.method,
-            success: extrinsic.success,
-            fee: exportFeeFromDepositEvent(extrinsic)?.toString()
+        let transfer = findTransferCall(extrinsic)
+        if (transfer != undefined) {
+            element.transfer = transfer
+        } else {
+            element.extrinsic = {
+                hash: extrinsic.extrinsic.hash.toString(),
+                module: extrinsic.extrinsic.method.section,
+                call: extrinsic.extrinsic.method.method,
+                success: extrinsic.success,
+                fee: exportFeeFromDepositEvent(extrinsic)?.toString()
+            }
         }
 
         await element.save()
     }
 }
+
+function findTransferCall(extrinsic: SubstrateExtrinsic): Transfer | null {
+    if (!isTransfer(extrinsic.extrinsic.method)) {
```

Comment:
success Transfer emit Transfer event, so it is handled at Transfers.ts. Add check `if (transfer.success) { return; }`